ui/dashboards/src/context/DashboardProvider/layout-slice.tsx

```diff
@@ -158,6 +164,32 @@ export function createLayoutSlice(layouts: LayoutDefinition[]): StateCreator<Lay
         [state.layouts[x], state.layouts[y]] = [yPanelGroup, xPanelGroup];
       });
     },
+
+    deletePanelGroup(groupIndex) {
+      set((state) => {
+        // remove group from state.layouts
+        const deletedPanelGroup = state.layouts.splice(groupIndex, 1);
+
+        // build an object that maps each panel to the groups it belongs
+        const panels = (get() as DashboardStoreState).panels;
+        const groups = get().layouts;
```

Comment:
Curious - why does `panels` need `as DashboardStoreState` but not `groups`?

Reply:
because panels live in another slice `panel-editing-slice`